service/meta.proto

```diff
@@ -39,55 +39,94 @@ message TokenLifetime {
   uint64 iat = 3;
 }
 
-// NeoFS session token.
-message SessionToken {
-  // Session token body
-  message Body {
-    // ID is a token identifier. valid UUIDv4 represented in bytes
-    bytes id = 1;
+// Context information for Session Tokens related to ObjectService requests
+message ObjectServiceContext {
+  // Object request verbs
+  enum Verb {
+    // Unknown verb
+    VERB_UNSPECIFIED = 0;
 
-    // OwnerID carries identifier of the session initiator.
-    neo.fs.v2.refs.OwnerID owner_id = 2;
+    // Refers to object.Put RPC call
+    PUT = 1;
 
-    // Verb is an enumeration of session request types
-    enum Verb {
-      // Unknown verb
-      VERB_UNSPECIFIED = 0;
+    // Refers to object.Get RPC call
+    GET = 2;
 
-      // Refers to object.Put RPC call
-      OBJECT_PUT = 1;
+    // Refers to object.Head RPC call
+    HEAD = 3;
 
-      // Refers to object.Get RPC call
-      OBJECT_GET = 2;
+    // Refers to object.Search RPC call
+    SEARCH = 4;
 
-      // Refers to object.Head RPC call
-      OBJECT_HEAD = 3;
+    // Refers to object.Delete RPC call
+    DELETE = 5;
 
-      // Refers to object.Search RPC call
-      OBJECT_SEARCH = 4;
+    // Refers to object.GetRange RPC call
+    RANGE = 6;
 
-      // Refers to object.Delete RPC call
-      OBJECT_DELETE = 5;
+    // Refers to object.GetRangeHash RPC call
+    RANGEHASH = 7;
+  }
+  // Verb is a type of request for which the token is issued
+  Verb verb = 1;
+  // Related Object address
+  neo.fs.v2.refs.Address address = 2;
+}
 
-      // Refers to object.GetRange RPC call
-      OBJECT_RANGE = 6;
+// Context information for Session Tokens related to Service requests
+message ContainerServiceContext {
```

Comment:
I would rather not add `ContainerServiceContext` until we figure out how neofs-node treats tokens for container requests. We can add it later without compatibility breaks.